src/test/java/LoginTests.java

```diff
@@ -108,6 +108,19 @@ public static void LoginWrongEmailTest () {
         System.out.println("==== This is our link text ==== " + link);
         driver.quit();
     }
+        @Test
+        public static void RegistrationNavigation() {
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example `public static void registrationNavigation()`